tests/conftest.py

```diff
@@ -20,12 +23,21 @@
 pytest.UID = audeer.uid()[:8]
 
 
+@pytest.fixture(scope='package', autouse=True)
+def register_single_folder():
+    if os.name != 'nt':
+        audbackend.register('single-folder', SingleFolder)
```

Comment:
Could you add a comment why it fails under Windows

Reply:
Windows is already giving me too much pain today.

Reply:
Hehe, ok, don't worry. I just thought you know as always ;)